prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/OllamaClient.kt

```diff
@@ -80,6 +80,10 @@ public class OllamaClient(
             )
         }.body<OllamaChatResponseDTO>()
 
+        if (response.error != null) {
+            throw IllegalArgumentException(response.error)
```

Comment:
@Newspicel I don't completely understand these changes, could you please explain this approach a little bit?